### PR TITLE
feat: add character controller

### PR DESCRIPTION
**Pull request by Rinder5**

### CODE REVIEW

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            _rb.velocity = new Vector3(0, Input.GetAxis("Jump") * JumpForce, 0);
+            _onGround = false;
+        }
+    }
+
+    private void OnCollisionStay()
+    {
+        _onGround = true;
+    }
+}
```
Comment (nicholaschuayunzhi) [inline Assets/Scripts/PlayerController.cs]: Will this enable the character to wall jump?

File: Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            _rb.velocity = new Vector3(0, Input.GetAxis("Jump") * JumpForce, 0);
+            _onGround = false;
+        }
+    }
+
+    private void OnCollisionStay()
+    {
+        _onGround = true;
+    }
```
Comment (nicholaschuayunzhi) [inline Assets/Scripts/PlayerController.cs:63]: Will this allow the character to wall climb/ jump?

**Comment by nicholaschuayunzhi** [comment]
Since, we need a prototype by tomorrow. Let's just use this and then work on fixing the bug.

Possible solution would be to detect ground with a base collider or implementing our own physics with raycasts.

File: Assets/Scripts/PlayerController.cs (hunk 2 above)
Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs:63]: yes.

Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs:63]: I've made an issue for fixing it: #6

**Comment by nicholaschuayunzhi** [comment]
Something that slipped my mind - Could you create a prefab for the character?

**Pull request merged**